src/core/defs.jl

```diff
@@ -132,17 +132,18 @@ description(def::DatumDef) = def.description
 
 unit(def::DatumDef) = def.unit
 
-# step_size(md::ModelDef) = step_size(indexvalues(md))
+function first_and_step(md::ModelDef)
+    keys::Vector{Int} = dim_keys(md, :time) # keys are, e.g., the times of the model runs
```

Comment:
I don't think we want to use these kind of type declarations on the left side of ``=``. I'm not even sure what they do, but I don't think they do what we want (namely ensure the type of ``keys``).